tap_qualtrics/__init__.py

```diff
@@ -394,8 +398,13 @@ def sync_survey_responses(config, state, stream):
                 "format": "csv",
                 "compress": True
             }
+            try:
+                records = get_survey_responses(survey_id, payload, config)
+            except Qualtrics404Error:
+                # Continue to the next survey_id if the current survey is
+                # not found.
+                break
```

Comment:
Should we log something here? Otherwise it's not obvious which responses to survey_id are being skipped